client/package.json

```diff
@@ -4,10 +4,6 @@
   "private": true,
   "dependencies": {
     "@craco/craco": "^6.2.0",
-    "@testing-library/jest-dom": "^5.11.4",
```

Comment:
removed Create React Apps testing packages to let root `package.json` handle the testing libraries.